### PR TITLE
Added new mbstring sniffs

### PR DESCRIPTION
**Pull request by mikeklein13**

### CODE REVIEW

**Comment by andyroo2000** [comment]
Nice work on your first sniff!!  🎉

**Comment by andyroo2000** [comment]
👍

File: Behance/Sniffs/Functions/FunctionMbStringSniff.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+   *                                        the token was found.
+   *
+   * @return void
+   */
+  public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
+
+    $tokens        = $phpcsFile->getTokens();
+    $token_content = $tokens[ $stackPtr ]['content'];
+
+    if ( in_array( $token_content, array_keys( self::MBSTRING_FUNCTION_MAP ) ) ) {
```
Comment (mikesherov) [inline Behance/Sniffs/Functions/FunctionMbStringSniff.php]: `if ( isset( self::MBSTRING_FUNCTION_MAP[ $token_content ] ) )`

O(1) vs. O(2n)

File: tests/Behance/Sniffs/Functions/FunctionMbStringTest.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        8  => 1,
+        9  => 1,
+        10 => 1,
+        11 => 1,
+        12 => 1,
+        13 => 1,
+        14 => 1,
+        15 => 1,
+        16 => 1,
+        17 => 0,
```
Comment (mikesherov) [inline tests/Behance/Sniffs/Functions/FunctionMbStringTest.php]: Don't need to specify zero, just the lines with > 1

**Comment by mikesherov** [comment]
👍

**Pull request merged**